lib/provider/data_types.go

```diff
@@ -70,7 +79,7 @@ type Volume struct {
 	// LunID the lun of volume
 	LunID string `json:"lunId,omitempty"`
 
-	// TargetIPAddresses list of target IP addresses for iscsi
+	// TargetIPAddresses list of target IP addresses for iscsi. Applicable for block storage only
```

Comment:
For this variable can you add prefix as `Block` as this is only for block, similarly you used `File` for FileNetworkMountAddress.

Reply:
can you please address following 
1- I think `Block` will be more readable as compare to Iscsi
2- Please aline comments as per variable i.e line number 82